sources/packages/web/src/components/common/NoticeOfAssessmentFormView.vue

```diff
@@ -150,9 +150,6 @@ export default defineComponent({
     };
 
     const loadNOA = async () => {
```

Comment:
The `loadNOA` is used under watch effect, so we can either revert the watch effect keep some condition under watch effect(e.g. any  of these props changing) to call the loadNOA.